src/systemd_ctypes/bus.py

```diff
@@ -16,8 +16,10 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import asyncio
+import ctypes
```

Comment:
I'd also prefer to keep the ctypes stuff in the binding layer.  We'd probably need to come up with a way to deal with that.

Reply:
We could add a helper function called `is_message_little_endian` to `src/systemd_ctypes/libsystemd.py`?

Reply:
Pushed up my suggestion :slightly_smiling_face: